### PR TITLE
Run periodic jobs against with openstack stable branches

### PR DESCRIPTION
**Pull request by h00130372**
Run periodic jobs against with openstack stable branches
for gophercloud, terraform-provier-openstack and rust.

Closes: theopenlab/openlab#267

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -76,6 +76,24 @@
...
+        - terraform-provider-openstack-acceptance-test-ocata:
+            branches: master
+        - terraform-provider-openstack-acceptance-test-pike:
+            branches: master
+        - terraform-provider-openstack-acceptance-test-queens:
+            branches: master
+        - terraform-provider-openstack-acceptance-test-rocky:
+            branches: master
+        - terraform-provider-openstack-acceptance-test-stein:
+            branches: master
```
Comment (kiwik) [inline zuul.d/projects.yaml]: please order by desc

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -232,3 +269,14 @@
...
+- project:
+    name: dtantsur/rust-openstack
+    periodic-10/22:
+      jobs:
+        - rust-openstack-acceptance-queens:
+            branches: master
+        - rust-openstack-acceptance-rocky:
+            branches: master
+        - rust-openstack-acceptance-stein:
+            branches: master
```
Comment (kiwik) [inline zuul.d/projects.yaml]: please order by desc

File: zuul.d/projects.yaml [hunk 3]
```diff
@@ -202,6 +220,25 @@
...
+        - gophercloud-acceptance-test-queens:
+            branches: master
+        - gophercloud-acceptance-test-pike:
+            branches: master
+        - gophercloud-acceptance-test-ocata:
+            branches: master
+        - gophercloud-acceptance-test-newton:
+            branches: master
+        - gophercloud-acceptance-test-mitaka:
+            branches: master
```
Comment (kiwik) [inline zuul.d/projects.yaml]: Please merge above gophercloud jobs (see line 122) into this gophercloud project

File: zuul.d/projects.yaml (hunk 1 above)
Comment (h00130372) [reply zuul.d/projects.yaml]: Done

File: zuul.d/projects.yaml (hunk 3 above)
Comment (h00130372) [reply zuul.d/projects.yaml]: Done. And I remove the ut test job.

File: zuul.d/projects.yaml (hunk 2 above)
Comment (h00130372) [reply zuul.d/projects.yaml]: Done.

**Pull request merged**